requirements.txt

```diff
@@ -7,5 +7,6 @@ ckantoolkit
 pytz
 babel
 -e git+https://github.com/ckan/ckanext-dcat.git#egg=ckanext-dcat
+-e git+https://github.com/open-data/ckanext-extendedactivity.git#egg=ckanext-extendedactivity
```

Comment:
please remove this, we're deploying extendedactivity from the deploy scripts now so we can track its commit id.